ynab_sdk/utils/clients/cached_client.py

```diff
@@ -11,23 +12,45 @@ class CachedClient(BaseClient):
 
     def __init__(self, config: CachedConfig):
         super().__init__(config)
-        self.redis = Redis(host=config.redis_host, port=config.redis_port, db=config.redis_db)
+        self.redis = Redis(host=config.redis_host, port=config.redis_port, db=config.redis_db, password=config.redis_pass)
+        self._redis_ttl: Union[int, None] = 3600
+        self._redis_prefix: str = 'YNAB_ep_'
+
+    @property
+    def cache_time_to_live(self) -> Union[int, None]:
+        return self._redis_ttl
+
+    @cache_time_to_live.setter
+    def cache_time_to_live(self, ttl_value: Union[int, None]) -> None:
+        if ttl_value and ttl_value > 0:
+            self.logger.error(f'cache_time_to_live: TTL value set to {ttl_value}')
+            self._redis_ttl = ttl_value
+        else:
+            self.logger.error(f'cache_time_to_live: 0, negative or None TTL value: {ttl_value}, set to None')
+            self._redis_ttl = None
+
+    def clear_cache(self) -> None:
+        keys_count: int = 0
+        for k in self.redis.scan_iter(''.join([self._redis_prefix, '*'])):
+            self.redis.delete(k)
+            keys_count += 1
+        self.logger.error(f'clear_cache: {keys_count} keys deleted from cache')
```

Comment:
Use logger.info since this is not an error

Reply:
Done